README.md

```diff
@@ -373,6 +373,19 @@ You can also view the results of an individual run in the UI as well.
   <img src="./readme_images/TonicValidate-Run.png" width="800">
 </picture>
 
+# Telemetry
```

Comment:
For the time being lets please add a note to the top of the README. This will be helpful for existing users and make them more aware.